source/conf-mongodb.rst

```diff
@@ -22,43 +25,45 @@ In |abbr.qan|, you can monitor |mongodb| metrics and |mongodb| queries. Run the
 Setting Up the Required Permissions
 ================================================================================
 
-For |mongodb| monitoring services to be
-able work in |qan|, you need to set up the |mongodb-exporter| user. This user
-should be assigned the |cluster-monitor| role for the |db.admin| database and
-the *read* role for the |db.local| database.
+For |mongodb| monitoring services to be able work in |abbr.qan|, you need to
+set up the |mongodb-exporter| user. This user should be assigned the
+|cluster-monitor| role for the |db.admin| database and the *read* role for the
+|db.local| database.
 
-The following example that you can run in the |mongodb| shell, adds the
-|mongodb-exporter| user and assigns the appropriate roles.
+The following is an example you can run in the |mongodb| shell, to add the
+|mongodb-exporter| user and assign the appropriate roles:
 
 .. _code.pmm.qan-mongodb.conf.essential-permission.setting-up.db.get-sibling-db.create-user:
 
+.. The following does not show up on live version
+
 .. include:: .res/code/db.get-sibling-db.create-user.txt
 
 .. seealso::
 
    Adding a |opt.mongodb-metrics| monitoring service
       :ref:`pmm-admin.add.mongodb-metrics`
+.. The above line does not render...
 
 .. _pmm.qan-mongodb.configuring.profiling.enabling:
 
-`Enabling Profiling <conf-mongodb.html#pmm-qan-mongodb-configuring-profiling-enabling>`_
-========================================================================================
+Enabling Profiling
```

Comment:
Specifying hyperlinks in headers manually is a sort of a trick. It allows us to have titles with human-readable URLs which will not change if we change the title text. It is important to avoid external broken links to our docs. BTW, there is a more natural way to specify links in RST, but for some reason, it makes titles to be rendered with the monospace font and ugly pink background. So these URLs are the least evil.

Reply:
So the root problem is that on the main webpage it's hyperlinked but when you click that link "Enabling Profling" it's not anchored to itself, but instead takes you back to the index which is not desired behavior.  The only one that works as you'd expect/hope is "Setting up the required permissions" above..  I think I treated this heading less as a "linkable bullet" but more as a heading similar to the "Supported versions of MongoDB" earlier on the page. I will update this shortly.